.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: php
 
+sudo: false
+
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 before_script:
-  - composer install --prefer-source --no-interaction
```

Comment:
Oh, I though the API rate were removed (see: https://github.com/composer/composer/issues/4884#issuecomment-195229989)